### PR TITLE
golang format tools

### PR DESCRIPTION
**Pull request by mattmoor-sockpuppet**
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -type f -name '*.go' -print))`
  `goimports -w $(find -name '*.go' | grep -v vendor)`

### CODE REVIEW

File: pkg/reconciler/v1alpha1/pipelinerun/resources/pipelinerunresolution_test.go [hunk 1]
```diff
@@ -328,7 +328,7 @@ func TestResolvePipelineRun(t *testing.T) {
 		),
 	))
 	providedResources := map[string]v1alpha1.PipelineResourceRef{
-		"git-resource": v1alpha1.PipelineResourceRef{
```
Comment (bobcatfish) [inline pkg/reconciler/v1alpha1/pipelinerun/resources/pipelinerunresolution_test.go]: o noice

**Comment by bobcatfish** [comment]
/lgtm
/approve
/ok-to-test

image

**Comment by vdemeester** [comment]
```
input_output_steps_test.go:74: error comparing post steps: {[]v1alpha1.TaskResourceBinding}[0->?]:
-: v1alpha1.TaskResourceBinding{Name: "test-output-2", ResourceRef: v1alpha1.PipelineResourceRef{Name: "resource2"}, Paths: []string{"/pvc/test-multiple-outputs/test-output-2"}}
+: <non-existent>
    {[]v1alpha1.TaskResourceBinding}[?->1]:
-: <non-existent>
+: v1alpha1.TaskResourceBinding{Name: "test-output-2", ResourceRef: v1alpha1.PipelineResourceRef{Name: "resource2"}, Paths: []string{"/pvc/test-multiple-outputs/test-output-2"}}
```

:thinking: This looks like a flaky unit tests :sweat:

**Pull request merged**